### ISSUE TITLE
Clientside name getter returns garbage for remote players

### ISSUE THREAD

**Issue by LennartF22** [altmp/altv-issues#1349]
### Description of the problem

Clientside name getter returns garbage for remote players.

### Reproduction steps

```js
alt.on('consoleCommand', (name, args) => {
    if (name === 'names') {
        alt.Player.all.forEach(player => {
            alt.log(`ID: ${player.id} Name: ${player.name}`);
        });
    }
});
```

```
[23:56:13] ID: 60 Name: �8���@��A
[23:56:13] ID: 61 Name: Lennart
```

### Expected behaviour

Clientside name getter shouldn't return garbage for remote players :)

### Operating system

Windows 10

### Version

9.0-rc2

**Comment by ThomasMarangoni** [comment]
Please add atleast a screenshot.

**Comment by FabianTerhorst** [comment]
are you sure you updated the module dll?

**Comment by LennartF22** [comment]
> are you sure you updated the module dll?

Yes

**Comment by C0kkie** [comment]
Fixed in  rc/9.0-rc4
